registry/index.adoc

```diff
@@ -0,0 +1,31 @@
+[id="registry-overview"]
+= {product-title} registry overview
+include::modules/common-attributes.adoc[]
+:context: registry-overview
+
+toc::[]
+
+{product-title} can build images from your source code, deploy them, and manage
+their lifecycle. To enable this, {product-title} provides an internal,
+integrated container image registry that can be deployed in your {product-title}
+environment to locally manage images. The _Registry Overview_ contains reference information and links for registries commonly used with OpenShift. It focuses on internal Image Registry.
```

Comment:
Should probably go ahead and unwrap these lines

Reply:
acked